INSTALL.md

```diff
@@ -487,7 +487,7 @@ In nginx this would be something like:
 ```
 location /.well-known/matrix/client {
     return 200 '{"m.homeserver": {"base_url": "https://<matrix.example.com>"}}';
-    add_header Content-Type application/json;
+    default_type "application/json";
```

Comment:
```suggestion
    default_type application/json;
```

Oh, and looking at the [sample nginx config](https://nginx.org/en/docs/example.html), I don't believe the double-quotes are necessary.